CHANGELOG.md

```diff
@@ -5,6 +5,105 @@ All notable changes to this project will be documented in this file.
+* build(deps): bump docker/login-action from 2.0.0 to 2.1.0 (#805, @dependabot[bot])
+* build(deps): bump github.com/google/go-cmp from 0.5.8 to 0.5.9 (#788, @dependabot[bot])
+* build(deps): bump github.com/sirupsen/logrus from 1.8.1 to 1.9.0 (#766, @dependabot[bot])
+* build(deps): bump github.com/spf13/cobra from 1.5.0 to 1.6.1 (#806, @dependabot[bot])
+* build(deps): bump github.com/spf13/viper from 1.12.0 to 1.13.0 (#790, @dependabot[bot])
+* build(deps): bump github.com/spf13/viper from 1.13.0 to 1.14.0 (#808, @dependabot[bot])
+* build(deps): bump github.com/stretchr/testify from 1.7.3 to 1.7.5 (#756, @dependabot[bot])
+* build(deps): bump github.com/stretchr/testify from 1.7.5 to 1.8.0 (#758, @dependabot[bot])
+* build(deps): bump github.com/stretchr/testify from 1.8.0 to 1.8.1 (#802, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.12 to 2.1.14 (#755, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.14 to 2.1.15 (#757, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.15 to 2.1.16 (#762, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.16 to 2.1.18 (#777, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.18 to 2.1.19 (#778, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.19 to 2.1.22 (#785, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.22 to 2.1.24 (#789, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.24 to 2.1.25 (#791, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.25 to 2.1.26 (#793, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.26 to 2.1.27 (#796, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.27 to 2.1.35 (#822, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.35 to 2.1.36 (#824, @dependabot[bot])
+* build(deps): bump github/codeql-action from 2.1.36 to 2.1.37 (#825, @dependabot[bot])
+* build(deps): bump golang.org/x/sys from 0.2.0 to 0.3.0 (#823, @dependabot[bot])
+* build(deps): bump golang.org/x/sys from 0.3.0 to 0.4.0 (#835, @dependabot[bot])
+* build(deps): bump golangci/golangci-lint-action from 3.2.0 to 3.3.0 (#807, @dependabot[bot])
+* build(deps): bump golangci/golangci-lint-action from 3.3.0 to 3.3.1 (#818, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.47.0 to 1.48.0 (#763, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.48.0 to 1.49.0 (#784, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.49.0 to 1.50.0 (#797, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.50.0 to 1.50.1 (#800, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.50.1 to 1.51.0 (#819, @dependabot[bot])
+* build(deps): bump google.golang.org/protobuf from 1.28.0 to 1.28.1 (#771, @dependabot[bot])
+* CHANGELOG.md: fix PR ref in the v0.10.0 release note (#753, @kaworu)
```

Comment:
nit: IMHO all the dependabot entries should be removed besides maybe Go version bump

Reply:
So far, we've kept them - as it is generated right now, each commit on the branch has a matching entry here. The script warns if there are commits without a PR for them.

I agree that this is noisy and probably should be collapsed - but I'd prefer it the release script would do that by itself: https://github.com/cilium/release/